neo/core/spiketrainlist.py

```diff
@@ -319,3 +343,51 @@ def multiplexed(self):
                 self._spike_time_array = np.hstack(spike_times) * self._items[0].units
                 self._channel_id_array = np.hstack(channel_ids)
         return self._channel_id_array, self._spike_time_array
+
+    @property
+    def t_start(self):
+        if "t_start" in self._spiketrain_metadata:
+            return self._spiketrain_metadata["t_start"]
+        else:
+            t_start_values = unique([item.t_start for item in self._items
+                                    if isinstance(item, SpikeTrain)])  # ignore proxy objects
```

Comment:
proxy objects also have t_start and t_stop properties. Would it be a problem to also include them here?

Reply:
Or do you plan a ProxySpikeTrainList object instead in the future?